dapp-oeth/src/components/layout.js

```diff
@@ -68,7 +68,18 @@ const Layout = ({
           content="https://ousd.com/images/share-twitter.png"
         />
       </Head>
-
+      <div
```

Comment:
Can we put these alerts behind a ENV var so that we don't have to submit a PR and deploy it everytime?

Reply:
Looking at this again, I guess it makes more sense to read from the contracts directly. This should never require deployment or ENV change

Reply:
I think we are over using ENV variables. The usage of const is also good :)